modules/coverage-report/src/test/java/org/jooby/jackson/JsonCustomMapperFeature.java

```diff
@@ -31,12 +32,12 @@
 
   @Test
   public void get() throws URISyntaxException, Exception {
-    request()
-        .get("/members")
-        .expect("[ {\n" +
-            "  \"id\" : 1,\n" +
-            "  \"name\" : \"pablo\"\n" +
-            "} ]");
+      request().get("/members").expect(value ->
+              assertEquals("[ {\n" +
```

Comment:
the `.expect` is part of Jooby testing code... a better option is to make the change there and not per each test